actions/global_actions.jsx

```diff
@@ -446,11 +446,12 @@ export function emitRemoteUserTypingEvent(channelId, userId, postParentId) {
     });
 }
 
-export function emitUserLoggedOutEvent(redirectTo = '/', shouldSignalLogout = true) {
-    // If the logout was intentional (as it should be if emitUserLoggedOutEvent is called),
-    // discard knowledge about having previously been logged in. This bit is otherwise used to
-    // detect session expirations on the login page.
-    LocalStorageStore.setWasLoggedIn(false);
+export function emitUserLoggedOutEvent(redirectTo = '/', shouldSignalLogout = true, userAction = true) {
```

Comment:
Not related to the PR but adding more than one flag as arguments seems odd to me. If we have to add another one we might be writing a lot of `trues` add `false` when calling this event.

Any thoughts on having this as an object more like 

```emitUserLoggedOutEvent({redirectTo, shouldSignalLogout, userAction});```

For me this seems cleaner as well when we call this as we can avoid the params we dont need to send and more redable when calling this functions.